src/Formatter/ClassNameFormatter.php

```diff
@@ -26,6 +26,11 @@ public function commandHandled($command)
         return 'Command succeeded: ' . get_class($command);
     }
 
+    public function propertiesUsed($command)
```

Comment:
I guess logger term should be implicit as all this stuff is made for logger.

What about `commandContext` ? To follow others methods names